sql/5_0_0-to-5_0_1_upgrade.sql

```diff
@@ -404,4 +404,9 @@ UPDATE categories_seq SET id = (select MAX(id) from categories);
 
 #IfNotRow2D list_options list_id apptstat option_id ^
 INSERT INTO list_options ( `list_id`, `option_id`, `title`, `seq`, `is_default`, `notes` ) VALUES ('apptstat','^','^ Pending',70,0,'FEFDCF|0');
+#EndIf
+
+#IfNotRow background_services name ccdaservice
+INSERT INTO `background_services` (`name`, `title`, `execute_interval`, `function`, `require_once`, `sort_order`) VALUES ('ccdaservice', 'C-CDA Node Service', 1, 'runCheck', '/ccdaservice/ssmanager.php', 95);
+ALTER TABLE `background_services` CHANGE `running` `running` TINYINT(1) NOT NULL DEFAULT '-1' COMMENT 'True indicates managed service is busy. Skip this interval.';
```

Comment:
I like the placing of this comment. Should also place it this in the database.sql table.